src/vstest.console/Processors/TestAdapterPathArgumentProcessor.cs

```diff
@@ -204,7 +214,7 @@ public ArgumentProcessorResult Execute()
         /// <returns> The list of test adapter and logger assemblies. </returns>
         internal virtual IEnumerable<string> GetTestAdaptersFromDirectory(string directory)
```

Comment:
Do we need this function now? I think we can remove it and we should add logic to find extension in testplatform.UpdateTestAdapterPaths( string runSettings )